plugins/modules/terraform_output.py

```diff
@@ -166,32 +121,34 @@ def main():
             binary_path=dict(type="path"),
             state_file=dict(type="path"),
         ),
-        required_if=[
-            ("format", "raw", ("name",))
-        ],
-        required_one_of=[
-            ("project_path", "state_file")
-        ]
+        required_if=[("format", "raw", ("name",))],
+        required_one_of=[("project_path", "state_file")],
     )
 
-    project_path = module.params.get("project_path")
-    bin_path = module.params.get("binary_path")
-    state_file = module.params.get("state_file")
-    name = module.params.get("name")
-    output_format = module.params.get("format")
+    project_path: Optional[str] = module.params.get("project_path")
+    bin_path: Optional[str] = module.params.get("binary_path")
+    state_file: Optional[str] = module.params.get("state_file")
+    name: Optional[str] = module.params.get("name")
+    output_format: str = module.params.get("format")
 
     if bin_path is not None:
         terraform_binary = bin_path
```

Comment:
could you please add the same check as the one in the terraform module line 329 to validate that the binary is existing ?